src/euphorie/upgrade/content/v1/20250402124248_add_content_type__choice/types.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<object name="portal_types">
+  <object meta_type="Conditional Dexterity FTI"
+          name="euphorie.choice"
+  />
+  <object meta_type="Conditional Dexterity FTI"
```

Comment:
Does this need to be a `Conditional Dexterity FTI`?
Or is an only addable in choices?

Reply:
I guess it could also be a regular FTI. Currently there is no restriction.